doc/users/next_whats_new/stem_orientation.rst

```diff
@@ -0,0 +1,24 @@
+Added *orientation* parameter for stem plots
+--------------------------------------------
+When creating stem plots, you can now pass in an *orientation* argument to
+`.Axes.stem` or `.pyplot.stem`.
+
+The default orientation produces stem lines that are vertically oriented:
+
+.. plot::
+
+    locs = np.linspace(0.1, 2 * np.pi, 25)
+    heads = np.cos(locs)
+
+    fig, ax = plt.subplots()
+    ax.stem(locs, heads)
+
+With the ``'horizontal'`` orientation, the stem lines will be horizontal:
+
+.. plot::
+
+    locs = np.linspace(0.1, 2 * np.pi, 25)
+    heads = np.cos(locs)
+
+    fig, ax = plt.subplots()
+    ax.stem(locs, heads, orientation='horizontal')
```

Comment:
More compact:

```suggestion
By default, stem lines that are vertical. This can be changed to horizontal
usting the *orientation* parameter:

.. plot::

    locs = np.linspace(0.1, 2 * np.pi, 25)
    heads = np.cos(locs)

    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(8, 4))
    ax1.stem(locs, heads)
    ax2.stem(locs, heads, orientation='horizontal')
```